cli/integration_tests/basic_monorepo/infer_pkg.t

```diff
@@ -12,7 +12,9 @@ Run a dry run
 Run a dry run in a directory
   $ cd packages/util
   $ ${TURBO} build --dry=json | jq .packages
-  []
+  [
+    "util"
+  ]
```

Comment:
This was accidentally merged in #3372, there was a legitimate failure, but it was missed because there were a lot of other unrelated failures that were safe to ignore in that PR.